client-v2/README.md

```diff
@@ -60,20 +116,66 @@ And to fix any issues that are automatically resolvable by eslint run:
 yarn lint-fix
 ```
 
-## Flow
-We are using flow for typing in Fronts V2. In order to install types for new
-libraries added run:
+| Uses        | For         | Config |
+| ------------|------------- |--- |
+| [Prettier](https://github.com/prettier/prettier)      | Anti-bikeshed Auto syntax formatting | [.prettierrc](.prettierrc)|
+| [.editorconfig](https://editorconfig.org/)      | Standard Editor formatting  |[.editorconfig](.editorconfig)|
+| [TSLint](https://palantir.github.io/tslint/)      | Typescript Linting | [tslint](tslint.json)|
 
-```bash
-yarn update-types
-```
+## Typescript
+We are using Typescript for typing in Fronts V2. 
 
-If you would like to overwrite types that are previously installed to newer
-versions that have been updated in the project pass the `--overwrite` flag to
-the above command.
+## File Structure
 
-If you want to run flow to check for type errors, run `yarn flow`.
+```
+src
+├── actions
+├── bundles
+├── components
+│   ├── CAPI
+│   ├── FrontsCAPIInterface
+│   ├── FrontsEdit
+│   ├── clipboard
+│   ├── inputs
+│   ├── layout
+│   └── util
+├── constants
+├── containers
+├── fixtures
+├── fonts
+│   ├── headline
+│   └── text
+├── images
+│   └── icons
+├── lib
+│   └── dnd
+├── reducers
+├── selectors
+├── services
+├── shared
+│   ├── actions
+│   ├── bundles
+│   ├── components
+│   ├── constants
+│   ├── fixtures
+│   ├── images
+│   ├── reducers
+│   ├── selectors
+│   ├── types
+│   └── util
+├── types
+└── util
+    └── sharedStyles
+```
 
-To check for flow coverage, run `yarn flow-coverage`.
+- Components, Actions, Reducers and Selectors are top level.
+    - The [App component](src/components/App.tsx) is the application entry point
+    - All reducers are combined in the [Root Reducer](src/reducers/rootReducer.ts) as per standard convention
+- `bundles` **what is the concept of a bundle?**
```

Comment:
@jonathonherbert do you reckon you could add a suggested change here just to give a simple overview of the key concepts?

Reply:
```suggestion
- `bundles`
    - A bundle exports a reducer and all of the related things a reducer needs to function in an app - selectors, actions and action names. It's a bit like an index.ts for a single redux module. This is especially useful when you're generating the actions, reducer and selectors rather than writing them manually, for example with the `createAsyncResourceBundle` utility in shared/util.
```

Reply:
Excellenté - thank @jonathonherbert